### ISSUE TITLE
Zoning facts contain hyphen/dash instead of underscore

### ISSUE THREAD

**Issue by mamoep** [brocade/ansible#83]
I wanted to gather the zoning facts to feed the active configuration back into the brocade_zoning_cfg module. Unfortunately some part of the facts still contain hyphens instead of underscore so trying to address the effective configuration from the facts fails.

Test was done with ansible tower 3.8.2, Ansible 2.9.19 and brocade collection 1.2.0.

**Playbook**
```yaml
  tasks:
  - name: get zoning facts
    brocade_facts:
      credential: "{{ credential }}"
      vfid: -1
      gather_subset:
        - brocade_zoning
    register: brocade_facts
    connection: local
  - name: show zoning
    debug:
      var: brocade_facts.ansible_facts.brocade_zoning
  - name: Add zone {{ create_zone }} to current active configuration {{ brocade_facts.ansible_facts.brocade_zoning.effective-configuration.cfg_name }}
    brocade_zoning_cfg:
      credential: "{{ credential }}"
      vfid: -1
      members_add_only: True
      cfgs:
        - name: "{{ brocade_facts.ansible_facts.brocade_zoning.effective-configuration.cfg_name }}"
          members:
            - "{{ create_zone }}"
      active_cfg: "{{ brocade_facts.ansible_facts.brocade_zoning.effective-configuration.cfg_name }}"
    connection: local
```
**Error**
`The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'effective'`

**Facts**
```json{
    "zoning_facts": {
        "failed": false,
        "changed": false,
        "ansible_facts": {
            "brocade_zoning": {
                "effective-configuration": {
                    "enabled_zone": {
                        "zone_type": "0",
                        "zone_name": "test_zone",
                        "member_entry": {
                            "entry-name": [
                                "12:12:12:12:12:12:12:1a",
                                "12:12:12:12:12:12:12:2a"
                            ]
                        }
                    },
                    "db_committed": "264",
                    "transaction_token": "0",
                    "checksum": "925c5efa18005123657273e63ebbb84c",
                    "default_zone_access": "noaccess",
                    "cfg_name": "cfg_test",
                    "db_chassis_wide_committed": "1276",
                    "cfg_action": "0",
                    "db_transaction": "0",
                    "db_avail": "1042754",
                    "db_max": "1045274"
                },
                "defined-configuration": {
                    "cfg": {
                        "member-zone": {
                            "zone-name": "test_zone"
                        },
                        "cfg-name": "cfg_test"
                    },
                    "zone": [
                        {
                            "zone-type": "0",
                            "member-entry": {
                                "entry-name": [
                                    "12:12:12:12:12:12:12:1a",
                                    "12:12:12:12:12:12:12:2a"
                                ]
                            },
                            "zone-name": "test_zone"
                        },
                        {
                            "zone-type": "0",
                            "member-entry": {
                                "entry-name": [
                                    "test_alias_2",
                                    "test_alias"
                                ]
                            },
                            "zone-name": "test_zone_2"
                        }
                    ],
                    "alias": [
                        {
                            "alias-name": "test_alias",
                            "member-entry": {
                                "alias-entry-name": "12:12:12:12:12:12:12:3a"
                            }
                        },
                        {
                            "alias-name": "test_alias_2",
                            "member-entry": {
                                "alias-entry-name": "12:12:12:12:12:12:12:2b"
                            }
                        }
                    ]
                }
            },
            "discovered_interpreter_python": "/usr/bin/python",
            "ssh_hostkeymust": true
        }
    },
    "changed": false,
    "_ansible_verbose_always": true,
    "_ansible_no_log": false
}```

**Comment by mamoep** [comment]
I found a workaround, addressing the variable works via ```"{{ brocade_facts.ansible_facts.brocade_zoning['effective-configuration'].cfg_name }}"```

Inconsistent structure should be fixed anyway.

**Comment by daniel-chung-broadcom** [comment]
Looking into the details and will follow up with you. Thanks.

**Comment by daniel-chung-broadcom** [comment]
pushed updated on documentation_attribute_content_change. Please let me know if any issues with the change.

**Comment by mamoep** [comment]
Update is looking good, no more dashes in the zoning facts.  Could you push this change to the collection?

**Comment by daniel-chung-broadcom** [comment]
Great. I'll keep you posted on the collection update.

**Comment by mamoep** [comment]
I want to use this with Ansible Automation Hub, aka Tower. I would really appreciate an updated collection.

**Comment by atos-simon-hill** [comment]
@daniel-chung-broadcom  any news on when this will get in to a new version of the collection please?

**Comment by mamoep** [comment]
According to infos from Brocade the Ansible module was abandoned.
If you are interested I created my own collection with all the fixes that are available in the official branches. Currently I find no time for further development.
https://galaxy.ansible.com/mamoep/brocade_fos

**Comment by dbond007** [comment]
Thanks,  I was thinking of doing the same. 
Probably should try and get some interested people to start up a community version and keep developing it.
I was thinking of doing that (given time), with HPE array phython sdk and ansible modules which they appear to have dropped / left to rot like brocade have.

**Comment by daniel-chung-broadcom** [comment]
Not sure where manoep has received information about being abandoned. 

dbond007, I think the best venue would be to use the existing brocade/ansible to post issues and collaborate. 

Atos-simon-hill, there are some issues being worked out. We will keep you posted.

**Comment by mamoep** [comment]
I got that information in summer from people at Brocade. The obvious lack of activity in this repository speaks for itself. I would be very happy if it is untrue and Brocade keeps on investing in this code. The last commit is from April, that was 8 months ago! When will we see some progress?

**Comment by marcusthordal** [comment]
The information you got may have been confused with the fact that the PyFOS development went into sustaining mode (due to lack of interest from customers).

Brocade continues to develop Ansible modules and provide collection updates along the way.
Repository activity does not always reflect ongoing development.

**Comment by prasad-valmeti-broadcom** [comment]
This is just to give an update. Plan is in progress to merge documentation branch changes into main/galaxy. Will update once the plan is finalized.

**Comment by prasad-valmeti-broadcom** [comment]
The issue is already addressed in Ansible FOS 1.3.3. If there are no further issues, please suggest if we can close the issue.
